### PR TITLE
Fix Rubocop offenses

### PR DESCRIPTION
**Pull request by archanaserver**
Depends on: https://github.com/theforeman/foreman_default_hostgroup/pull/62

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Rails:
```
Comment (ekohl) [inline .rubocop.yml:1]: The Rails cops are useful, so I'd keep them in.

Comment (archanaserver) [reply .rubocop.yml:1]: Yeah, just updated another change to fix that Rails issue https://github.com/theforeman/foreman_default_hostgroup/pull/62/commits/805f3da3dbd688bf9d34ae57e8b7e6e7281187d6

File: lib/tasks/foreman_default_hostgroup.rake [hunk 2]
```diff
@@ -12,7 +12,7 @@ namespace :test do
 end
 
 namespace :foreman_default_hostgroup do
-  task :rubocop do
+  task rubocop: :environment do
```
Comment (ekohl) [inline lib/tasks/foreman_default_hostgroup.rake]: RuboCop doesn't depend on the Rails environment
```suggestion
  task :rubocop do
```

File: test/unit/default_hostgroup_test.rb [hunk 3]
```diff
@@ -39,7 +39,7 @@ def setup_hostgroup_matchers
   end
 
   def setup_host_and_facts
-    raw = JSON.parse(File.read(File.expand_path(File.dirname(__FILE__) + '/facts.json')))
+    raw = JSON.parse(File.read(File.expand_path("#{File.dirname(__FILE__)}/facts.json")))
```
Comment (ekohl) [inline test/unit/default_hostgroup_test.rb]:
```suggestion
    raw = JSON.parse(File.read(File.join(__dir__, 'facts.json")))
```

File: test/unit/default_hostgroup_test.rb [hunk 4]
```diff
@@ -39,7 +39,7 @@ def setup_hostgroup_matchers
   end
 
   def setup_host_and_facts
-    raw = JSON.parse(File.read(File.expand_path(File.dirname(__FILE__) + '/facts.json')))
+    raw = JSON.parse(File.read(File.join(__dir__, 'facts.json")))
```
Comment (ekohl) [inline test/unit/default_hostgroup_test.rb]:
```suggestion
    raw = JSON.parse(File.read(File.join(__dir__, 'facts.json')))
```

**Comment by archanaserver** [comment]
I have changed the title, because now this PR contains more offenses fix.

**Comment by archanaserver** [comment]
@ekohl does it need any other changes? would you mind taking a look here also at the parent PR of this one https://github.com/theforeman/foreman_default_hostgroup/pull/62

**Review by ekohl — APPROVED** [review]

**Pull request merged**